CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.1 - 6-5-2024
+
+### Fixed
+
+- exit code `2` if there are no parse errors or validation failures, but there are lint warnings.
+  - exit code `0` means there were no concerns found at all.
```

Comment:
For completeness, maybe add that it exits `1` for parse errors or validation failures